### PR TITLE
Small improvements to builder output

### PR DESCRIPTION
**Pull request by ManneW**
Just some small suggestions on improvements to the builder output that I came across when trying out the builder.

### CODE REVIEW

File: builder.sh [hunk 1]
```diff
@@ -294,12 +294,12 @@ function run_build() {
...
             for j in {1..3}; do
-                bashio::log.info "Start upload $i"
+                bashio::log.info "Start upload of $i (attempt #$j/3)"
                 if docker push "$i" > /dev/null 2>&1; then
-                    bashio::log.info "Upload success after $j"
+                    bashio::log.info "Upload succeeded on attempt #$j"
                     break
                 fi
-                bashio::log.warning "Upload fail on $j"
+                bashio::log.warning "Upload failed on attempt #$j"
```
Comment (frenck) [inline builder.sh:302]:
```suggestion
                bashio::log.warning "Upload failed on attempt #${j}"
```

File: builder.sh [hunk 2]
```diff
@@ -294,12 +294,12 @@ function run_build() {
     if [ "$DOCKER_PUSH" == "true" ]; then
         for i in "${push_images[@]}"; do
             for j in {1..3}; do
-                bashio::log.info "Start upload $i"
+                bashio::log.info "Start upload of $i (attempt #$j/3)"
                 if docker push "$i" > /dev/null 2>&1; then
-                    bashio::log.info "Upload success after $j"
+                    bashio::log.info "Upload succeeded on attempt #$j"
```
Comment (frenck) [inline builder.sh:299]:
```suggestion
                    bashio::log.info "Upload succeeded on attempt #${j}"
```

File: builder.sh [hunk 3]
```diff
@@ -294,12 +294,12 @@ function run_build() {
     if [ "$DOCKER_PUSH" == "true" ]; then
         for i in "${push_images[@]}"; do
             for j in {1..3}; do
-                bashio::log.info "Start upload $i"
+                bashio::log.info "Start upload of $i (attempt #$j/3)"
```
Comment (frenck) [inline builder.sh:297]:
```suggestion
                bashio::log.info "Start upload of $i (attempt #${j}/3)"
```

**Comment by ManneW** [comment]
Thanks, @frenck ! I've updated based on the comments now 🙂

**Pull request merged**